webpack/assets/javascripts/react_app/components/SubmitCancelButtons/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import Actions from '../common/forms/Actions';
+import {
+  onSubmitClick,
+  onCancelClick,
+  onMountButtons,
+} from '../../redux/actions/user/submitCancelButtons';
+
+export class SubmitCancelContainer extends React.Component {
+  componentDidMount() {
+    this.props.mountDispatch();
+  }
+  render() {
+    const {
+      data: { cancelPath },
+      submitting,
+      disabled,
+      windowLocation,
+      cancelDispatch,
+      submitDispatch,
+    } = this.props;
+    return (
+      <Actions
+        submitting={submitting}
+        disabled={disabled}
+        onCancel={() => {
+          cancelDispatch();
+          windowLocation.replace(cancelPath);
+        }}
+        onSubmitClick={submitDispatch}
+      />
+    );
+  }
+}
+
+const mapStateToProps = (state) => {
+  const { submitCancelButtons } = state;
+  return {
+    submitting: submitCancelButtons.submitting || false,
+    disabled: submitCancelButtons.disabled || false,
+    windowLocation: submitCancelButtons.windowLocation || window.location,
+  };
+};
+
+const mapDispatchToProps = dispatch =>
```

Comment:
Check this out, a simpler way to attach action-dispatchers:
https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/BreadcrumbBar/index.js#L21